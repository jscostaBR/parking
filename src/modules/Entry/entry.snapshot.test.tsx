import * as React from 'react'
import { Entry } from '.'
import renderer from 'react-test-renderer'

describe('render Entry', () => {
  it('render correctly', () => {
    const tree = renderer.create(<Entry/>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})

export = {}
