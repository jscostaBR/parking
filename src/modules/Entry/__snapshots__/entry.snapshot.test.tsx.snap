// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Entry render correctly 1`] = `
<div
  className="sc-bczRLJ kknKxP"
>
  <form
    onSubmit={[Function]}
  >
    <label
      className="sc-gsnTZi fiLyGA"
    >
      <span>
        NÃºmero da placa:
      </span>
      <input
        name="entry"
        onBlur={[Function]}
        onChange={[Function]}
        onFocus={[Function]}
        onMouseDown={[Function]}
        onPaste={[Function]}
        placeholder="AAA-0000"
        value=""
      />
    </label>
    <button
      className="sc-hKMtZM sc-eCYdqJ kDbrgC kIlOPt"
      disabled={true}
    >
      CONFIRMAR ENTRADA
    </button>
  </form>
</div>
`;
